troika_tb/utils/run_snpit.py

```diff
@@ -9,14 +9,20 @@ def run_snpit(vcf):
     # print(vcf)   
     v = pathlib.Path(vcf)
     if v.exists():
-        cmd = f"snpit --input {v}"
+        cmd = f"snpit-run.py --input {v}"
         p = subprocess.run(cmd, shell = True, capture_output = True, encoding = "utf-8")
+        print (p)
```

Comment:
Can we remove this print statement?

Reply:
Yes we can comment it out. I think it existed in the old version but was commented out.